sys/arm64/arm64/pmap.c

```diff
@@ -6193,6 +6193,44 @@ pmap_copy_pages(vm_page_t ma[], vm_offset_t a_offset, vm_page_t mb[],
 		} else {
 			b_cp = (char *)PHYS_TO_DMAP_LEN(p_b + b_pg_offset, cnt);
 		}
+#if __has_feature(capabilities)
```

Comment:
Since the function is essentially the same, does it make sense to have common implementation for these and switch the bcopy/bcopynocap with a flag? Or is it easier to merge this way maybe.

Reply:
I tried using a helper function with a bool argument with wrappers and I think it looks ok.

Reply:
Looks good to me.